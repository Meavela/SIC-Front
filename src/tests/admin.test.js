import React from 'react';
import renderer from 'react-test-renderer';
import Admin from '../components/admin';
import { BrowserRouter } from 'react-router-dom';

it('admin page correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
            <Admin />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });