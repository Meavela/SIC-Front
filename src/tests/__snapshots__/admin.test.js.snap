// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin page correctly 1`] = `
<p>
  You have to be connected as admin to view this page
</p>
`;
